arrays/arrays.c

```diff
@@ -157,12 +157,17 @@ void arr_append(Array *arr, char *element) {
  *****/
 void arr_remove(Array *arr, char *element) {
 
-  // Search for the first occurence of the element and remove it.
-  // Don't forget to free its memory!
-
-  // Shift over every element after the removed element to the left one position
-
-  // Decrement count by 1
+  int position = 0;
+  for (int i = 0; i < arr->count; i++)
+  {
+    if (arr->elements[i] == element)
+    {
+      position = i;
+      arr->elements[i] = NULL;
+      // Don't forget to free its memory!
+      free(arr->elements[i]);
+    }
+  }
```

Comment:
you might be leaving a hole here, remember to shift all the items right of where you removed the item!